app/assets/javascripts/app/views/notifications_view.js

```diff
@@ -70,7 +70,8 @@ app.views.Notifications = Backbone.View.extend({
 
     [allNotes, typeNotes, headerBadge].forEach(function(element){
       element.text(function(i, text){
-        return parseInt(text) + change });
+        return Math.max(parseInt(text) + change, 0);
```

Comment:
Theorically, yes. As the notifications count is not deduced from an actual list, but computed taking account user actions. It's not likely to happen, but it's possible. The best solution would be to have a model for notifications user-side.